src/pybind/feat/online_feature_pybind.cc

```diff
@@ -0,0 +1,93 @@
+// pybind/feat/online_feature_pybind.cc
+
+// Copyright 2019   Microsoft Corporation (author: Xingyu Na)
+
+// See ../../../COPYING for clarification regarding multiple authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// THIS CODE IS PROVIDED *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION ANY IMPLIED
+// WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR PURPOSE,
+// MERCHANTABLITY OR NON-INFRINGEMENT.
+// See the Apache 2 License for the specific language governing permissions and
+// limitations under the License.
+
+#include "feat/online_feature_pybind.h"
+
+#include "feat/feature-mfcc.h"
+#include "feat/feature-fbank.h"
+#include "feat/online-feature.h"
+
+using namespace kaldi;
+  
+template <class Feature>
+void online_base_feature(py::module& m, const std::string& feat_type) {
+  py::class_<OnlineGenericBaseFeature<Feature>, OnlineBaseFeature>(m, feat_type.c_str())
+      .def(py::init<const typename Feature::Options&>())
+      .def("Dim", &OnlineGenericBaseFeature<Feature>::Dim)
```

Comment:
I think `Dim` and other methods do not need to be re-wrapped here.
They are already wrapped in their parent classes, i.e., OnlineFeatureInterface.